src/scalarstats.jl

```diff
@@ -240,6 +309,14 @@ function _zscore_chksize(X::AbstractArray, μ::AbstractArray, σ::AbstractArray)
     end
 end
 
+
+"""
+    zscore!([Z,] X, μ, σ)
+
+Compute the z-scores of an array `X` with mean `μ` and standard deviation `σ`. If
```

Comment:
I think he's asking for an English definition of z-score(s), rather than assuming the reader/user knows what a z-score is.